ArcGISRuntimeSDKQt_QMLSamples/Geometry/GeodesicOperations/GeodesicOperations.qml

```diff
@@ -0,0 +1,122 @@
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// [Legal]
+
+import QtQuick 2.6
+import Esri.ArcGISRuntime 100.3
+import Esri.ArcGISExtras 1.1
+
+Rectangle {
+    id: rootRectangle
+    clip: true
+    width: 800
+    height: 600
+
+    property real scaleFactor: System.displayScaleFactor
+    property string distanceText
+
+    MapView {
+        id: mapView
+        anchors.fill: parent
+
+        // create a graphics overlay
+        GraphicsOverlay {
+
+            // add a marker graphic for NYC
+            Graphic {
+                id: nycGraphic
+                Point {
+                    x: -73.7781
+                    y: 40.6413
+                    spatialReference: SpatialReference { wkid: 4326 }
+                }
+                symbol: markerSymbol
+            }
+
+            // add a graphic for the path
+            Graphic {
+                id: pathGraphic
+                symbol: pathSymbol
+            }
+
+            // add an end marker graphic
+            Graphic {
+                id: destinationGraphic
+                symbol: markerSymbol
+            }
+        }
+
+        // create a map with the imagery basemap
+        Map {
+            BasemapImagery {}
+        }
+
+        // handle the mouse click
+        onMouseClicked: {
+            // re-project the point to match the NYC graphic
+            var destination = GeometryEngine.project(mouse.mapPoint, nycGraphic.geometry.spatialReference);
+
+            // update the destination graphic
+            destinationGraphic.geometry = destination;
+
+            // create line with start/end points
+            var polylineBuilder = ArcGISRuntimeEnvironment.createObject("PolylineBuilder", {spatialReference: destination.spatialReference});
+            polylineBuilder.addPoints([nycGraphic.geometry, destinationGraphic.geometry]);
+            var polyline = polylineBuilder.geometry;
+
+            // densify the path as a geodesic curve and show it with the path graphic
+            var maxSegmentLength = 1;
+            var unitOfMeasurement = ArcGISRuntimeEnvironment.createObject("LinearUnit", {linearUnitId: Enums.LinearUnitIdKilometers});
+            var curveType = Enums.GeodeticCurveTypeGeodesic;
+            var pathGeometry = GeometryEngine.densifyGeodetic(polyline, maxSegmentLength, unitOfMeasurement, curveType);
+
+            // update the graphic
+            pathGraphic.geometry = pathGeometry;
+
+            // calculate the path's geodetic length
+            distanceText = Math.round(GeometryEngine.lengthGeodetic(pathGeometry, unitOfMeasurement, curveType));
```

Comment:
Is there a reason to round here instead of showing fractional distance values in kilometers?